### ISSUE TITLE
PostgreSQL support (postgres)

### ISSUE THREAD

**Issue by samip5** [monoxane/vxconnect#1]
Would that be possible?

**Comment by monoxane** [comment]
Postgres is definitely possible, the persistence is modular and designed to handle any backend, it's just a matter of time and effort to do it :)

**Comment by samip5** [comment]
Yeah, I was actually looking at it and it's really good that it's modular. :)
